src/Raven.Studio/typescript/common/generalUtils.ts

```diff
@@ -3,6 +3,12 @@ import pluralizeHelpers = require("common/helpers/text/pluralizeHelpers");
 import moment = require("moment");
 import d3 = require("d3");
 
+interface FormatDetailedDatabaseNameProps {
+    databaseName: string;
+    nodeTag: string;
+    location?: databaseLocationSpecifier;
```

Comment:
I think location has node tag + shard, right?

Reply:
Yes, but if the database is not sharded or is in single shard view ("dbName$shard") then location is null